dashboard/app/views/admin_reports/funometer.html.haml

```diff
@@ -1,100 +1,77 @@
 - content_for :head do
   %script{src:'https://www.google.com/jsapi'}
 
-%h1
-  Fun-O-Meter Stats
+- if @stats.present?
+  %h1
+    Fun-O-Meter Stats
 
-%h2
-  Overall Percentage:
-  %span= number_to_percentage(@overall_percentage, precision: 2)
+  %h2
+    Overall Percentage:
+    %span= number_to_percentage(@stats['overall'], precision: 2)
 
-%br
+  %br
 
-= render 'funometer_percentages_by_day', :percentages_by_day => percentages_by_day
+  -# Renable the graphs after reconciling the different object types.
+  -# = render 'funometer_percentages_by_day', :percentages_by_day => @stats['ratings_by_day']
 
-%br
+  %br
 
-%h2
-  Fun-O-Meter by Script
+  %h2
+    Fun-O-Meter by Script
 
-%table
-  %tr
-    - @script_headers.each do |header|
-      %th
-        %span= header
-  - @script_ratings.each do |script|
+  %table
     %tr
-      %td
-        %span
-          = link_to funometer_by_script_path(:script_id => script.script_id) do
-            = script.script_id
-      %td
-        %span
-          = link_to script_url(script.script_id) do
-            = script.name
-      %td
-        %span= number_to_percentage(script.percentage, precision: 2)
-      %td
-        %span= script.cnt
+      - ['Script ID', 'Script Name', 'Percentage', 'Count'].each do |header|
+        %th
+          %span= header
+    - @stats['ratings_by_script'].each do |script|
+      %tr
+        %td
+          %span
+            = link_to funometer_by_script_path(:script_id => script[0]) do
+              = script[0]
```

Comment:
Actually, it would make this an array of hashes. Based on offline conversation, I'm adding a TODO (in the cron file) to think about making this change.